templates/project/.github/workflows/qa.yaml.twig

```diff
@@ -42,9 +42,25 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-
-      - name: Psalm
-        uses: docker://vimeo/psalm-github-actions
+{% if project.repository.name in ['SonataDoctrineMongoDBAdminBundle', 'sandbox'] %}
```

Comment:
There should be another way to configure this I guess from `projects.yaml`.

Reply:
We only need to install PHP for these 2 projects? 🤔

Reply:
😅 fixed, thanks.

Reply:
IMO it could be better if we copy the setup php from tests: https://github.com/sonata-project/dev-kit/blob/3ef9afc9183800096720dde6860892d7bf481aa9/templates/project/.github/workflows/test.yaml.twig#L60

Php for every project and a exception for sandbox and mongodb

Reply:
I agree with @jordisala1991

Reply:
I thought at first about it, but `services` variable is available? I thought it was only under `branch`.

Reply:
It was/is under `branch` 👍